api/integrations/github/github.py

```diff
@@ -0,0 +1,146 @@
+import datetime
+import logging
+import typing
+
+import requests
+from django.conf import settings
+
+from features.models import FeatureState, FeatureStateValue
+from integrations.github.client import generate_token
+from integrations.github.constants import GITHUB_API_URL
+from integrations.github.models import GithubConfiguration
+from webhooks.webhooks import WebhookEventType
+
+logger = logging.getLogger(__name__)
+
+
+def post_comment_to_github(installation_id, owner, repo, issue, body):
+    try:
+        token = generate_token(
+            installation_id,
+            settings.GITHUB_APP_ID,
+        )
+
+        url = f"{GITHUB_API_URL}repos/{owner}/{repo}/issues/{issue}/comments"
+        headers = {
+            "Accept": "application/vnd.github.v3+json",
+            "Authorization": f"Bearer {token}",
+        }
+
+        payload = {"body": body}
+        response = requests.post(url, json=payload, headers=headers)
```

Comment:
We should add timeout to all the requests that we are making

Reply:
Timeouts added.